src/metadataExt.ts

```diff
@@ -123,17 +124,17 @@ export class MetadataModuleExt {
             const translator = createTranslator(kpiMap, visWidgetMap);
             const updatedContent = updateContent(analyticalDashboard, translator, filterContext);
             const dashboardTitle = this.getDashboardName(analyticalDashboard.meta.title, options.name);
-            const duplicateDashboard: IAnalyticalDashboard = {
+            const { tags, summary, isProduction, deprecated } = dashboardDetails.analyticalDashboard.meta;
+            const duplicateDashboard = {
                 ...dashboardDetails,
                 analyticalDashboard: {
                     ...dashboardDetails.analyticalDashboard,
-                    content: {
-                        filterContext,
-                        layout: { ...updatedContent.layout },
-                        widgets: [...updatedContent.widgets],
-                    },
+                    content: this.getDashboardDetailObject(updatedContent, filterContext),
                     meta: {
-                        ...dashboardDetails.analyticalDashboard.meta,
+                        tags,
```

Comment:
You could simplify this using lodash's [`omit`](https://lodash.com/docs/4.17.15#omit) saying what you do NOT want to include:

```ts
const duplicateDashboard = {
  ...dashboardDetails,
  content: this.getDashboardDetailObject(updatedContent, filterContext),
  ...(omit(dashboardDetails.analyticalDashboard.meta, ["identifier", "uri", "author", "created", "updated", "contributor"])),
  title: dashboardTitle,
}
```

This has the added benefit, that you cannot "forget" to copy something important, you just remove the stuff you do not want there. I believe that here, it is worse if you don't include something you should have, that if you include something you shouldn't have.